pallets/sto/src/lib.rs

```diff
@@ -162,7 +162,7 @@ decl_storage! {
         /// All fundraisers that are currently running. (ticker, fundraiser_id) -> Fundraiser
         Fundraisers get(fn fundraisers): double_map hasher(blake2_128_concat) Ticker, hasher(twox_64_concat) u64 => Option<Fundraiser<T::Balance, T::Moment>>;
         /// Total fundraisers created for a token
-        FundraiserCount get(fn fundraiser_count): map hasher(twox_64_concat) Ticker => u64;
+        FundraiserCount get(fn fundraiser_count): map hasher(blake2_128_concat) Ticker => u64;
```

Comment:
Can you elaborate on this one?

Reply:
Tickers are user inputs. Albeit it will be an expensive attack to do due to high protocol fees, I'd rather play it safe.